far/changelog

```diff
@@ -1,3 +1,7 @@
+MKadaner 28.01.2018 15:50:33 -0800 - build 5134
```

Comment:
Please drop the ` - build 5134` part (since it's not changed).

Reply:
Sure